src/app/editor-common/components/type-select/type-select.component.ts

```diff
@@ -120,12 +118,9 @@ export class InputTypeSelectComponent extends DirectiveBase implements ControlVa
         this.onTouched = fn;
     }
 
-    setDisabledState(isDisabled: boolean) {
-        if (isDisabled) {
-            this.skipOnChange = true;
-            this.form.controls["type"].disable();
-            this.form.controls["isItemOrArray"].disable();
-            this.form.controls["items"].disable();
-        }
+    setDisabledState(isDisabled: boolean): void {
+        this.readonly = isDisabled;
+        this.skipOnChange = true;
```

Comment:
Change this to work without skipOnChange method (also this appears in writeValue method).